pkg/kubelet/lifecycle/handlers_test.go

```diff
@@ -259,3 +258,453 @@ func TestRunHandlerHttpFailure(t *testing.T) {
 		t.Errorf("unexpected url: %s", fakeHTTPGetter.url)
 	}
 }
+
+func TestNoNewPrivsRequired(t *testing.T) {
+	for _, testCase := range []struct {
+		input    *v1.Pod
+		expected bool
+	}{
+		{
+			input: &v1.Pod{
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
```

Comment:
would be nice to test with two containers, where the first one doesn't have `AllowPrivilegeEscalation` and second does